src/components/ActionBar/src/ActionBarButton.vue

```diff
@@ -47,15 +47,18 @@ import { MLoading } from '@square/maker/components/Loading';
 // TODO: refactor the code below so it's shared with Button component
 
 function getContrast(chromaBg, targetChromaFg) {
-	if (!targetChromaFg || chroma.contrast(chromaBg, targetChromaFg) < 4.5) {
-		const isLight = chromaBg.luminance() > 0.32;
+	const accessibleContrast = 4.5;
```

Comment:
Can you please move these ActionBarButton refactors to a separate PR? They seem to have nothing to do with the detecting Safari.

Reply:
These aren't refactors, it's resolving lint issues regarding no magic numbers: https://eslint.org/docs/rules/no-magic-numbers